### PR TITLE
remove platform configuration settings and re-phrase

### PR DESCRIPTION
**Pull request by BeateRixen**
I have removed all configuration descriptions from the user documentation which must be done from the Management tenant. 
Moreover I have added some standard phrasing "Contact your Operations team  for support/details/configuration", as suggest by Jane, in all places where we mention that only the management tenant has permission for some process.
@rssag All information that has been entirely removed must go into a separate chapter in the Operations guide (we already talked about that). Please create a draft and we can then walk through the content together.

### CODE REVIEW

File: content/users-guide/administration-bundle/managing-permissions.md [hunk 1]
```diff
@@ -340,10 +340,6 @@ Permissions are grouped into the following categories:
 |Device control|Permissions to remote control devices.
 |Full access|Complete access to the associated devices, mainly to simplify configuration.
 
-{{< c8y-admon-info >}}
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-permissions.md:343]: @rssag This must be explained in the UI section of the operations guide with some context on the permissions page.

File: content/users-guide/enterprise-tenant-bundle/managing-tenants.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ In the {{< management-tenant >}}, you also find a column with information on the
 
 When the subtenant is created, it gets an auto-generated ID, which cannot be changed. Also, it is automatically provisioned with a first, administrative user ("Administrator's username"). This administrator can create other users and set their permissions. The first user cannot be deleted to prevent you from locking yourself out.
 
-From the {{< management-tenant >}}, you can enable other tenants to create subtenants. To do so, check **Allow creation of subtenants** in the tenant editor.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/managing-tenants.md:125]: @rssag Must be moved to the Ops guide

File: content/users-guide/enterprise-tenant-bundle/managing-tenants.md [hunk 3]
```diff
@@ -272,12 +272,7 @@ The request throttling mechanism is only enabled when both HTTP properties (limi
 Rate limiting can be an effective countermeasure against threats like brute force login attempts, API abuse and request flooding thus reducing the number of malicious/unwanted traffic. This helps in protecting against DoS (Denial of Service) attacks and saving the available bandwidth for legitimate requests.
 {{< /c8y-admon-important >}}
 
-It is also possible to customize the buffer size for the CEP queue and the data broker queue for a particular tenant. This can be done from the {{< management-tenant >}} by using the following subtenant custom fragments:
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/managing-tenants.md:275]: @rssag Must be moved to the Ops guide

File: content/users-guide/enterprise-tenant-bundle/ms-data-broker.md [hunk 4]
```diff
@@ -28,16 +28,7 @@ In summary, to work with the microservice-based data broker, the following requi
 <a name="enabling-ms-data-broker"></a>
 ###  To enable the microservice-based data broker
 
-1. In the {{< management-tenant >}}, navigate to **Administration** > **Tenants** > **Subtenants**, and select the tenant that will serve as the source of the data connector.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/ms-data-broker.md:31]: @rssag Must be moved to the Ops guide

File: content/users-guide/enterprise-tenant-bundle/support-user-access.md [hunk 5]
```diff
@@ -32,35 +31,13 @@ If deactivated globally, support user access can still be enabled by a subtenant
...
 {{< /c8y-admon-info >}}
 
-The duration of the active support request can be globally configured in the {{< management-tenant >}} (default is 24 hours), see [Administration > Platform configuration settings > Support user](/users-guide/administration/#config-support-users).
+The duration of the active support request can be globally configured in the {{< management-tenant >}} (default is 24 hours). Contact your Operations team on configuring this setting towards your needs.
 
 Each new support request will prolong the support duration for the specified number of hours. After the last support request in a subtenant has expired or has been actively disabled by the user, the support user access for the subtenant will immediately be disabled (if not activated globally).
 
 Details on the status of support requests and support user access for a tenant can be found in the **Properties** tab of the tenant, see [Managing tenants](/users-guide/enterprise-tenant#managing-tenants).
-
-### To log in as support user
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/support-user-access.md:46]: @rssag Must be moved to the Ops guide

File: content/users-guide/enterprise-tenant-bundle/ms-data-broker.md [hunk 6]
```diff
@@ -28,16 +28,7 @@ In summary, to work with the microservice-based data broker, the following requi
...
-2. In the **Applications** tab:
-    * Subscribe the tenant to the data broker feature (`feature-broker` application), if it is not already subscribed.
-    * Subscribe the tenant to the data broker agent (`databroker-agent-server` application).
-
-![Databroker-Agent-Server](/images/users-guide/enterprise-tenant/et-new-data-broker-agent.png)
-
-{{< c8y-admon-info >}}
-Tenants subscribing to a data connector don't need the data broker agent subscription.
-{{< /c8y-admon-info >}}
+The microservice-based data broker must be enabled from the {{< management-tenant >}}. Contact your Operations team on any further support.
```
Comment (rssag) [inline content/users-guide/enterprise-tenant-bundle/ms-data-broker.md:31]:
```suggestion
The microservice-based data broker must be enabled from the {{< management-tenant >}}. Contact your Operations team for further support.
```

File: content/users-guide/enterprise-tenant-bundle/support-user-access.md [hunk 7]
```diff
@@ -22,8 +22,7 @@ Support user access can either be:
 * Activated for all subtenants by default.
 * Deactivated for all subtenants, but explicitly be enabled by a user for their tenant.
 
-This is configured globally in the *{{< management-tenant >}}*, see [Administration > Platform configuration settings > Support user](/users-guide/administration/#config-support-users).
-
+This is configured globally in the *{{< management-tenant >}}*. Contact your Operations team on configuring the settings towards your needs.
```
Comment (rssag) [inline content/users-guide/enterprise-tenant-bundle/support-user-access.md:25]:
```suggestion
This is configured globally in the *{{< management-tenant >}}*. Contact your Operations team on how to configure the settings according to your needs.
```

**Review by rssag — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Airdroplet Hi Iarek, could you please confirm that all items which we want to delete from the user documentation and move to the Ops documentation are the correct ones? We would like to merge this PR before the rebuild and will then create the new chapter in the Ops guide in 2or 3 sprints. Ticket is here: https://cumulocity.atlassian.net/browse/MTM-54694. This is a bit asynchronous but we have plenty of time until the 2024 release. For the rebuild, we would like to get the content clean.

**Review by Airdroplet — APPROVED** [review]

**Comment by Airdroplet** [comment]
Generally looks ok, with one open question from my perspective, since we removing management tenant related descriptions, do we already have a place in Ops guide for those?

**Comment by BeateRixen** [comment]
> 

@Airdroplet Please see my comment above :-). We will create a new chapter in the Operations guide including all UI configurations.

**Pull request merged**